designs/administrator/future-openshift/saved-searches/index.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Saved filters
+
+The Search page will be updated to allow users to save searches.
+
+## General flow
+
+![1](img/saved1.png)
+* If no filters have been saved, the **Saved filters** dropdown will have a small empty state.
+
+![2](img/saved2.png)
+* Users can save filters by clicking the **Save filter** button at the end of the filter chip row.
+
+![3](img/saved3.png)
+* Users will be able to enter a name for the saved filter in a modal.
+
+![4](img/saved4.png)
+* The **Saved filters** dropdown will update to show the saved filter name since it is the currently applied filter.
+
+![5](img/saved5.png)
+* Users can click the **Saved filters** dropdown to see all their saved searches.
+* Through the dropdown, users may also choose to edit their saved searches.
+
+![6](img/saved6.png)
+* In the **Edit saved filters** modal, users are able to edit the name of any previously saved filter.
+
+![7](img/saved7.png)
```

Comment:
that checkmark should be blue right?

Reply:
it also looks like PF would implement this with the input stretching the full width to the right to meet the edit icons

Reply:
@beanh66 do you know if there is a convention update planned to focus on when action buttons should be disabled or not? I don't see one now and I think this has always been a topic that isn't fully settled - if a button should be disabled OR enabled that triggers an alert with a description of what needs to be fixed before the user can proceed. Maybe I'm overthinking it?

Reply:
Updated based on your comments @matthewcarleton ! And yeah, I think we've had this conversation in past and settled on what @itsptk described above – disable if it's clear what you need to fix, but if it's out of view or otherwise hard to know, enabled so they can get the appropriate error.